src/DataFixtures/UserFixtures.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\User;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+
+
+class UserFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        $user = new User();
+        $user->setUsername('John')->setEmail("John123@gmail.com")->setRoles(array('ROLE_USER'))->setRegisterDate(new \DateTime())->setEnabled(1)->setPlainPassword("JohnPassword");
```

Comment:
When I said one line, I meant for the syntax ``set()->set()...``
But it makes a line too long so you can write it like this : 
```php
        $user
            ->setUsername('John')
            ->setEmail("John123@gmail.com")
```